python/tests/integration/test_methods.py

```diff
@@ -254,9 +245,47 @@ def test_it_runs_inline_query(sdk: mtds.LookerSDK, queries: TQueries):
         raise AssertionError("png format failed to return an image")
 
 
+@pytest.mark.usefixtures("remove_test_looks")
+def test_crud_look(sdk: mtds.LookerSDK, looks):
+    """Test creating, retrieving, updating and deleting a look."""
+    for l in looks:
+        request = create_query_request(l["query"][0], "10")
+        query = sdk.create_query(request)
+
+        look = sdk.create_look(
+            ml.WriteLookWithQuery(
+                title=l.get("title"),
+                description=l.get("description"),
+                deleted=l.get("deleted"),
+                is_run_on_load=l.get("is_run_on_load"),
+                public=l.get("public"),
+                query_id=query.id,
+                space_id=l.get("space_id") or str(sdk.me().personal_space_id),
+            )
+        )
+
+        assert isinstance(look, ml.LookWithQuery)
+        assert look.title == l.get("title")
```

Comment:
Is `obj.get("...")` the reference syntax for accessing properties of an object? That seems to be an anti-pattern

Reply:
`obj.get("property")` returns `None` if no second argument is provided. If we did `obj["property"]` we'd get a `KeyError` when the property doesn't exist

Reply:
At the same time, some of these properties are required so perhaps they should be accessed using the latter

Reply:
good note about the accessor error. I didn't consider that. I think for the required fields we should use `foo.bar` notation unless it becomes a hassle to check the property status.

Reply:
fwiw, we used the `.get` notation everywhere else throughout the tests.  In this case, `looks` is a list of dictionaries. The notation we can use is either `obj[key]` or `obj.get(key[, default])` (https://docs.python.org/3/library/stdtypes.html?highlight=dict%20get#dict.get). It wouldn't introduce any hassle to use `obj[key]` but I might need to go back and change all the other tests for consistency if we're going down this route. Based on the docs the only required parameters for `create_look` are `model` and `view` in `query`.

Reply:
I think `obj[key]` is slightly better than `obj.get(key...)` unless the semantics of failed gets changes

Reply:
Separate PR FTW

Reply:
Perfect, agreed. I'll leave these as is for now and create a separate PR first chance I get.